azure-pipelines-integration.yml

```diff
@@ -23,49 +23,49 @@ jobs:
         _repoName: "dotnet/format"
         _targetSolution: "format.sln"
         _branchName: "main"
-        _sha: "1ec992c6038af9d42499d3ac88c3fd65d0c2f6ed"
+        _sha: "452f6263277533da0af2b557003256d98fbecc27"
```

Comment:
❔ Why update all of these?

Reply:
Habit. Although I see these repos are now on some very *preview* sdk versions

Reply:
If updating the dependency broke formatting for the previous SHA, doesn't that mean some users could see dotnet-format break for the same change?